export/securedrop_export/disk/cli.py

```diff
@@ -296,40 +472,25 @@ def mount_volume(self, volume: Volume) -> MountedVolume:
             return MountedVolume.from_volume(volume, mountpoint)
 
         else:
-            logger.info("Mount volume at default mountpoint")
-            return self._mount_at_mountpoint(volume, self._DEFAULT_MOUNTPOINT)
+            try:
+                logger.info("Mount volume in /media/user using udisksctl")
+                output = subprocess.check_output(
+                    ["udisksctl", "mount", "-b", f"/dev/mapper/{volume.mapped_name}"]
```

Comment:
Nit: Suggest long arguments for readability:

```suggestion
                    ["udisksctl", "mount", "--block-device", f"/dev/mapper/{volume.mapped_name}"]
```